src/modules/clearAll.js

```diff
@@ -0,0 +1,19 @@
+import localGet from './storage.js';
+
+const clearCompleted = () => {
```

Comment:
- [ ] OPTIONAL
Marking a task as completed is not updating in the local storage. Kindly ensure that the task object's values for completed is updated upon user actions